### ISSUE TITLE
deprecated since Symfony 4.2

### ISSUE THREAD

**Issue by xorgxx** [leogout/SeoBundle#9]
hi have try your bundle and i get error deprecated : 
```
   ›     $treeBuilder = new TreeBuilder();
    ›     $rootNode = $treeBuilder->root('leogout_seo');
```
are you plan to fix?

Thank

**Comment by tugrul** [comment]
This problem looks like easy to fix by documentation. 

https://symfony.com/doc/current/components/config/definition.html#defining-a-hierarchy-of-configuration-values-using-the-treebuilder

```php
$treeBuilder = new TreeBuilder();
$rootNode = $treeBuilder->root('leogout_seo');
```
to
```php
$treeBuilder = new TreeBuilder('leogout_seo');
```

Please fix this problem because it is going to be not working at Symfony 5.

**Comment by leogout** [comment]
Hello, as you can see on this repository, no activity has been reported since 2017. This was a fun project to build but I don't have the time nor the patience to go back to PHP and Symfony 4 to improve it. I don't plan on supporting Symfony 5 as well.
If you want to bring modificaitons to the codebase, feel free to create a PR.

**Comment by tugrul** [comment]
@leogout I created pull request. 

Can you publish this version to composer repository?

Thanks

**Comment by leogout** [comment]
Thanks for your PR, it's been merged and tagged to v1.2.1. It's available for packagist as well:
https://packagist.org/packages/leogout/seo-bundle
